File: IronVault/IronVault.Model/Models/Nutricionist.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace IronVault.Model.Models
{
    public partial class Nutricionist
    {
        public int NutricionistId { get; set; }

        public string Ime { get; set; } = null!;

        public string Prezime { get; set; } = null!;

        public string Email { get; set; } = null!;

        public string BrojTelefona { get; set; } = null!;

        public string? Slika { get; set; }
    }
}

File: IronVault/UI/ironvault_desktop/lib/models/nutricionist.dart
import 'package:json_annotation/json_annotation.dart';

part 'nutricionist.g.dart';

@JsonSerializable()
class Nutricionist {
  int? nutricionistId;
  String? ime;
  String? prezime;
  String? email;
  String? brojTelefona;
  String? slika;

  Nutricionist({this.nutricionistId});

  factory Nutricionist.fromJson(Map<String, dynamic> json) =>
      _$NutricionistFromJson(json);

  /// Connect the generated [_$PersonToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$NutricionistToJson(this);
}
